pkg/network/config/config.go

```diff
@@ -110,6 +110,12 @@ type Config struct {
 	// Currently Windows only
 	HTTPMaxRequestFragment int64
 
+	// SSLAsyncHandshakeWindow is the duration between a SSL_do_handshake() and tcp_sendmsg() calls
+	// to have a valid correlation between the ssl context with the struct sock matching the same pid/tid
+	// only valid for openssl async implementation
+	// unit : nanoseconds
+	SSLAsyncHandshakeWindow uint64
```

Comment:
Better add NS suffix to the field name instead of the comment

Reply:
I changed the type  to time.Duration